bluzelled/inc/http/server/ServiceManager.h

```diff
@@ -0,0 +1,42 @@
+#ifndef KEPLER_SERVICEMANAGER_H
+#define KEPLER_SERVICEMANAGER_H
+
+#include "server.hpp"
+
+#include <boost/thread.hpp>
+
+namespace  http
```

Comment:
@rich This is the only file that uses namespaces. We should put them everywhere or remove from this one